src/app/pages/storage/volumes/volume-status/volume-status.component.ts

```diff
@@ -646,9 +653,9 @@ export class VolumeStatusComponent implements OnInit, OnDestroy {
     };
   }
 
-  onClickEdit(pk: string): void {
-    const diskForm = this.modalService.openInSlideIn(DiskFormComponent);
-    diskForm.inIt(pk);
+  onClickEdit(disk: Disk): void {
```

Comment:
I get empty form when I click edit in pool status page.

Reply:
Still doesn't work for me. It doesn't work even for the old code. See if you can use a different attribute besides `devname` when querying for disk.